syntax = "proto2";

message ForceVentilateIncubatorMessage {
    required uint32 duration = 2;
}
